app/src/main/AndroidManifest.xml

```diff
@@ -125,6 +125,7 @@
 
         <activity
             android:name=".achievements.AchievementsActivity"
+            android:configChanges="orientation|keyboardHidden|screenSize"
             android:label="@string/Achievements" />
```

Comment:
This approach is not suggested, please use savedInstanceState method instead.

Reply:
@neslihanturan Okay, I will make the changes 👍